Runtime/API.cs

```diff
@@ -90,22 +90,34 @@ public static async UniTask<LoginResponse> CreateUser(LoginRequest loginRequest,
 
         public static async UniTask<HttpResponse> RequestEmailAuth(string emailAddr, string userId, CancellationToken token = default)
         {
-            var result = await apiClient.Get<HttpResponse>($"/users/{userId}/verify-email?email={emailAddr}");
+            var result = await apiClient.Get($"/users/{userId}/verify-email?email={emailAddr}");
             if (!result.IsSuccess)
             {
                 Debug.Log($"RequestEmailAuth returned error {result.errorMsg}");
             }
+            return result;
+        }
+
+        public static async UniTask<HttpResponse> PreValidateEmail(string emailAddr, string userId, CancellationToken token = default)
+        {
+            PreValidateEmailRequest request = new();
+            request.email = emailAddr;
+            var result = await apiClient.Post<HttpResponse>($"/users/{userId}/email/pre-validate", request);
+            if (!result.IsSuccess)
+            {
+                Debug.Log($"PreValidateEmail returned error {result.errorMsg}");
+            }
             return result.body;
         }
```

Comment:
we need to remove this

Reply:
as we discussed, perhaps better to move this to the Demo, so we can include it in the testing.

Ideally also use a configuration file for public key and secret key. Be sure to use the secret key for PreValidateEmail